### PR TITLE
Add phind - resolves #39

### LINKED ISSUE #39: [Looking for Contributor] New Provider: Phind.com
in #32 we added Provider modules to make it easy to add more chats. if you're new here and want an easy way to contribute, now it's a lot easier! 

please claim this and send a PR for us to review!

this work is split out from #13

### PR DESCRIPTION
**Pull request by seanoliver**
Added Phind!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,8 +30,8 @@ Yes and no:
 | Claude 2 | ☑️ | **Login Tip**: Use manual email login and remember to type (not paste) the login code! ([#56](https://github.com/smol-ai/menubar/issues/56)) |
 | Local Models (via [OobaBooga](https://github.com/oobabooga/text-generation-webui)) | ☑️ | Requires Local Setup |
 | HuggingChat | ☑️ |  |
-| Perplexity | ☐ | Coming Soon — [Submit a PR](https://github.com/smol-ai/menubar/issues/35)! |
-| Phind | ☐ | Coming Soon — [Submit a PR](https://github.com/smol-ai/menubar/issues/39)! |
+| Perplexity | ☑️ |  |
```
Comment (swyxio) [inline README.md:33]: good catch

File: index.js [hunk 2]
```diff
@@ -34,6 +34,7 @@ const providers = {
 	OobaBooga: require('./providers/oobabooga'),
 	Smol: require('./providers/smol'),
 	HuggingChat: require('./providers/huggingchat'),
+	Phind: require('./providers/phind'),
 	Perplexity: require('./providers/perplexity'),
```
Comment (swyxio) [inline index.js:38]: rearranging for subjective order of readiness/ease of use/quality

```suggestion
	Perplexity: require('./providers/perplexity'),
	Phind: require('./providers/phind'),
	HuggingChat: require('./providers/huggingchat'),
	OobaBooga: require('./providers/oobabooga'),
	Smol: require('./providers/smol'),
```

File: interface.js [hunk 3]
```diff
@@ -13,6 +13,7 @@ const providers = {
 	OobaBooga: require('./providers/oobabooga'),
 	Smol: require('./providers/smol'),
 	HuggingChat: require('./providers/huggingchat'),
+	Phind: require('./providers/phind'),
 	Perplexity: require('./providers/perplexity'),
```
Comment (swyxio) [inline interface.js:17]:
```suggestion
	Perplexity: require('./providers/perplexity'),
	Phind: require('./providers/phind'),
	HuggingChat: require('./providers/huggingchat'),
	OobaBooga: require('./providers/oobabooga'),
	Smol: require('./providers/smol'),
```

**Review by swyxio — APPROVED** [review]

**Comment by lgandecki** [comment]
there is no "Describe your technical task" textarea after you send the first message. I fixed that in #92 .
excited to have both phind and perplexity in the menubar! productivity x5

**Comment by lgandecki** [comment]
heads up, that ^ fixed the input in the followup question so should be merged in here, but the follow up click is still missing

**Comment by lgandecki** [comment]
I think it clicks the right button, but for some reason it doesn't work for the followup question, this is what I get in a browser when I go go the phind.com to test (paste) this manually

```javascript
// execute this before:
  
var inputElement = document.querySelector('textarea[placeholder*="Describe your technical task"]');
if (!inputElement) {
	inputElement = document.querySelector('textarea[placeholder*="Send message"]');
}

function simulateUserInput(element, text) {
  var nativeTextAreaValueSetter = Object.getOwnPropertyDescriptor(window.HTMLTextAreaElement.prototype, "value").set;
  var event = new Event('input', { bubbles: true});

  nativeTextAreaValueSetter.call(inputElement, text);
  inputElement.dispatchEvent(event);
}

simulateUserInput(inputElement, "hello")
        
// and then:
var keyEvent = new KeyboardEvent('keydown', {key: ' ', bubbles: true});
inputElement.dispatchEvent(keyEvent);

// add a space to the input value
inputElement.value = inputElement.value + ' ';

// simulate keyup event
var keyupEvent = new KeyboardEvent('keyup', {key: ' ', bubbles: true});
inputElement.dispatchEvent(keyupEvent);

// click the submit button
var buttonElement = document.querySelector('button[type="submit"]');
buttonElement.click();
```
```
undefined
agent:1 Uncaught (in promise) DOMException: Document is not focused.
```

**Comment by lgandecki** [comment]
I just pushed a fix that finds and clicks the correct button for the follow up questions - also, don't worry, it didn't take me 7 hrs ;) I came back to this during brakes from work..  with that.. I believe the phind functionality is ready! 🍾

**Comment by swyxio** [comment]
appreciated!

**Comment by seanoliver** [comment]
Ah this is such a help — thanks @lgandecki!

**Pull request merged**